local-data-platform/real_world_use_cases/near_data_lake/config/sample_queries/near_transaction.py

```diff
@@ -0,0 +1,8 @@
+from collections import namedtuple
+
+Transaction = namedtuple('Transaction', ['query', 'desc'])
```

Comment:
@mrutunjay-kinagi can we please move this to local_data_platform.__init__.py

Reply:
Done